### ISSUE TITLE
Unable to import my Postman collection (Paid User)

### ISSUE THREAD

**Issue by tobiashochguertel** [AntonShuvaev/jet-client-support#57]
**Describe the bug**

When I select the `import Postman collection` button from the`add` menu, it shows me the dialog for importing a postman collection. I select in the dialog the `Collection file` and click then on the button `Import`, but then nothing happens. I also tried to import the collection without `Enable synchroniyation` or without `Folder description as common path prefix`  but that doesn't change the behavior - nothing happens.

![CleanShot 2024-03-30 at 09 34 52@2x](https://github.com/AntonShuvaev/jet-client-support/assets/3332669/e6cf02a9-b802-4425-b6ab-19a715ffd4e9)

**Steps to reproduce**

**Expected behavior**
Jetclient imports my collection.

**Environment information:**
- OS:

```
Software:

    System Software Overview:

      System Version: macOS 14.3.1 (23D60)
      Kernel Version: Darwin 23.3.0
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled

Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Chip: Apple M1 Max
      Total Number of Cores: 10 (8 performance and 2 efficiency)
      Activation Lock Status: Enabled
```

- IDE version:

![CleanShot 2024-03-30 at 09 37 44@2x](https://github.com/AntonShuvaev/jet-client-support/assets/3332669/c5114630-6ed6-44a9-bc15-574b8957d69e)

```
IntelliJ IDEA 2024.1 RC (Ultimate Edition)
Build #IU-241.14494.158, built on March 21, 2024
Licensed to Tobias Hochgürtel
Subscription is active until January 30, 2025.
Runtime version: 17.0.10+8-b1207.12 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.3.1
GC: G1 Young Generation, G1 Old Generation
Memory: 8192M
Cores: 10
Metal Rendering is ON
Registry:
  debugger.new.tool.window.layout=true
  ide.experimental.ui=true
  terminal.new.ui=true
  editor.minimap.enabled=true
Non-Bundled Plugins:
  com.jetbrains.plugins.ini4idea (241.14494.240)
  io.github.fstaudt.helm (0.8.2)
  LiveTemplatePreview (0.3)
  com.intellij.plugins.macoskeymap (241.13688.16)
  awesome.console (0.1337.12)
  com.tabnine.TabNine (1.67.0)
  freeriders.mag.Modular-Architecture-Generator (0.1.2-SNAPSHOT)
  com.paperetto.dash (3.3)
  name.kropp.intellij.makefile (241.14494.150)
  MultiHighlight (3.1.0)
  com.karateca.jstoolbox (1.10)
  org.intellij.plugins.hcl (241.14494.150)
  com.intellij.apacheConfig (241.14494.240)
  com.steve.plugins.autoscroll (1.1)
  com.samvtran.plugins.macosforallkeymap (2.0.0)
  com.jinsihou.react.snippets (1.1.1)
  com.jetbrains.edu (2024.3-2024.1-455)
  com.andrey4623.rainbowcsv (2.0.2)
  org.jetbrains.plugins.go-template (241.14494.150)
  com.redhat.devtools.intellij.telemetry (1.1.0.52)
  org.plugin.dot.id (1.5.3)
  dev.meanmail.plugin.nginx-intellij-plugin (2022.1.1)
  com.intellij.properties.bundle.editor (241.14494.150)
  com.bluewaitor.tsReact (1.4.0)
  dev.pnbarx.idea.treecolor (1.2.2)
  hunspell (241.14494.241)
  com.intellij.commander (241.14494.158)
  Groovy Console Plugin (0.1.5)
  net.ashald.envfile (3.4.2)
  DevKit (241.14494.237)
  String Manipulation (9.12.0)
  com.github.rmatafonov.cucumber-scenarios-indexer (1.1.0)
  gherkin (241.14494.150)
  com.intellij.aqua (241.14494.241)
  com.adgainai.springbooturlfullpath (1.0.3)
  com.chainstaysoftware.assertions2assertj (1.0.09)
  com.intellij.guice (241.14494.158)
  com.github.jonathanlocke.intellij.easymapmac (1.0.5)
  MavenRunHelper (4.27.1-IJ2022.2)
  io.conceptive.quarkus-intellij-plugin (1.3.1)
  org.asciidoctor.intellij.asciidoc (0.41.12)
  com.chuntung.plugin.gistsnippet (1.1.0)
  net.seesharpsoft.intellij.plugins.csv (3.3.0-241)
  com.intellij.ml.llm (241.14494.240)
  com.github.copilot (1.5.1.5285)
  uk.co.ben-gibson.remote.repository.mapper (4.4.0)
  mobi.hsz.idea.gitignore (4.5.2)
  com.intellij.bigdatatools.core (241.14494.240)
  com.haulmont.jpab (241.14494.158)
  com.github.bjansen.intellij.pebble (v0.11.0)
  JSTestDriver Plugin (202.5)
  org.mapstruct.intellij (1.6.1)
  dev.turingcomplete.intellijdevelopertoolsplugins (4.1.1)
  com.jetbrains.packagesearch.intellij-plugin (241.0.8)
  ru.adelf.idea.dotenv (2024.1)
  LivePlugin (0.8.6 beta)
  net.labymod.intellij.singlehotswap (2.6)
  com.clutcher.comments_highlighter (2024.1.11)
  com.intellij.tide (241.14494.241)
  mdx.js (241.14494.140)
  izhangzhihao.rainbow.brackets (2024.2.2-241)
  com.intellij.lang.jsgraphql (241.14494.150)
  io.jetclient (2024.1.4-241)
  com.github.lppedd.idea-return-highlighter (0.8.3)
  kemoke.net.cssmodule (1.1)
  com.dmarcotte.handlebars (241.14494.150)
  io.plugin.tsnode (2022.11.14-1)
  com.mallowigi (95.0.0)
  com.intellij.bigdatatools.kafka (241.14494.158)
  org.jetbrains.plugins.docker.gateway (241.14494.189)
Kotlin: 241.14494.158-IJ
```

- plugin version:

![CleanShot 2024-03-30 at 09 38 47@2x](https://github.com/AntonShuvaev/jet-client-support/assets/3332669/9b8e67d2-1aee-49de-b572-5523d95f0735)

```
2024.1.4-241
```

**Comment by AntonShuvaev** [comment]
Hello @tobiashochguertel ,

Thank you for reporting this issue. Could you please check if your Postman collection is version 2.1.0? Additionally, it would be very helpful if you could create a simple collection with any request, export it, and then try to import this new collection.

**Comment by tobiashochguertel** [comment]
```

**Comment by tobiashochguertel** [comment]
![CleanShot 2024-03-30 at 10 18 05@2x](https://github.com/AntonShuvaev/jet-client-support/assets/3332669/a7b485f5-d87a-41ae-8508-082289d5d884)

```
  "info": {
    "_postman_id": "0f2bfc62-8bba-4566-b51a-4f77e498f94d",
    "name": "Quiz API Service",
    "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
    "description": {
      "content": "",
      "type": "text/plain"
    }
  }
```

**Comment by tobiashochguertel** [comment]
A simple collection works... with setup Pre-request script and Authentication from the postman.

[simple.postman_collection.json](https://github.com/AntonShuvaev/jet-client-support/files/14811357/simple.postman_collection.json)

![CleanShot 2024-03-30 at 10 22 10@2x](https://github.com/AntonShuvaev/jet-client-support/assets/3332669/d3a96610-6585-4173-a2dc-f5a80e9d07b8)

Helpful would be a way to see a debug window or why it can't import my complex collection.

**Comment by AntonShuvaev** [comment]
Thank you for providing those details. I've found that some issues can occur during the import when certain properties are null. Normally, this would trigger a red error message in the dialog, as shown in the screenshot provided. Just to confirm, you didn't encounter any error messages?

![Image](https://github.com/AntonShuvaev/jet-client-support/assets/25157138/5b2b5ec1-eeff-4a28-922c-19ece7cc022a)

**Comment by tobiashochguertel** [comment]
No error message in read when I click on `import`:

it looks after one or more clicks on `import` as this:

![CleanShot 2024-03-30 at 11 30 20@2x](https://github.com/AntonShuvaev/jet-client-support/assets/3332669/2d79d5c0-701e-4fdf-ba82-3cc8dd814fc2)

**Comment by AntonShuvaev** [comment]
Thank you for the confirmation. I'll investigate further.

**Comment by tobiashochguertel** [comment]
When you give me an email address, I can share my collection with you. It's not for public thought. The import via OpenAPI3 worked well.

**Comment by AntonShuvaev** [comment]
You can send it to support@jetclient.io, thanks!

**Comment by tobiashochguertel** [comment]
The Mail is out.

**Comment by AntonShuvaev** [comment]
Fixed it in version 2024.1.5.

**Comment by light-developer** [comment]
@AntonShuvaev I want export to postman collection, please support, tks
